core/optaplanner-core-impl/src/test/java/org/optaplanner/core/impl/heuristic/move/CompositeMoveTest.java

```diff
@@ -80,6 +80,23 @@ void doMove() {
         verify(c, times(1)).doMove(scoreDirector);
     }
 
+    @Test
+    void doMoveOnly() {
```

Comment:
I appreciate this new test - however, it is identical to the `doMove` test.
In that case, the other test should make sure that some undo move is actually returned.